packages/fxa-auth-server/bin/key_server.js

```diff
@@ -105,7 +107,8 @@ function run(config) {
             Password,
             config,
             customs,
-            zendeskClient
+            zendeskClient,
+            subhub
```

Comment:
Moved `subhub` out here so we can call its `close()` when shutting down the server.